github/GithubException.py

```diff
@@ -38,35 +39,35 @@ class GithubException(Exception):
     Some other types of exceptions might be raised by underlying libraries, for example for network-related issues.
     """
 
-    def __init__(self, status, data, headers):
+    def __init__(self, status: int, data: Any, headers: Optional[Dict[str, str]]):
         super().__init__()
         self.__status = status
         self.__data = data
         self.__headers = headers
-        self.args = [status, data, headers]
+        self.args = (status, data, headers)
```

Comment:
type of `Exception.args` is `Tuple[Any, ...]` so mypy will raise a error for this